tests/integration/test_pairwise_significance.py

```diff
@@ -571,6 +571,19 @@ def test_pairwise_cat_x_mr_gender_x_all_pets_owned(self):
             ),
         )
 
+    def test_pairwise_significance_indices(self):
+        slice_ = Cube(OL.CAT_X_MR_GENDER_X_ALL_PETS_OWNED).partitions[0]
+
+        assert slice_.column_percentages.tolist() == [
+            [75.0, 20.0, 60.0],
+            [25.0, 80.0, 40.0],
+        ]
+
+        assert slice_.pairwise_indices.tolist() == [
+            [(), (0, 2), ()],
+            [(1,), (), (1,)],
+        ]
+
```

Comment:
I'd add a case with the 2 level of significance

Reply:
done